NEWS

```diff
@@ -1,9 +1,117 @@
 python-scrapinghub release notes
 ================================
 
+Version 1.9.0
+-------------
+(release on Nov 2nd, 2016)
```

Comment:
why not to include date in the header, seems to be common practice

https://github.com/scrapy/scrapy/blob/master/docs/news.rst
https://github.com/kennethreitz/requests/blob/master/HISTORY.rst

I like requests approach `<version> (<date>)` the most - short and clean